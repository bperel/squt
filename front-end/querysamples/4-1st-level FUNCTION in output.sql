SELECT co.ContractID AS ID, CONCAT(p.firstName, p.lastName) AS PersonFullName
FROM ContractID co
INNER JOIN Person p ON co.personInChargeID = p.ID
WHERE co.isActive = 1;