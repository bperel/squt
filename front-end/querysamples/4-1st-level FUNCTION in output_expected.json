{
   "Functions" : {
      "PersonFullName" : {
         "to" : "OUTPUT",
         "name" : "concat",
         "alias" : "PersonFullName"
      }
   },
   "Tables" : {
      "Person" : {
         "p" : {
            "OUTPUT" : {
               "firstName" : {
                  "PersonFullName" : "firstName"
               },
               "lastName" : {
                  "PersonFullName" : "lastName"
               }
            }
         }
      },
      "Contractid" : {
         "co" : {
            "OUTPUT" : {
               "ContractID" : {
                  "-1" : "ID"
               }
            },
            "CONDITION" : {
               "personInChargeID" : {
                  "p.ID" : "JOIN_TYPE_STRAIGHT"
               },
               "isActive" : {
                  "1" : "JOIN_TYPE_STRAIGHT"
               }
            }
         }
      }
   }
}